test/models/test_albef.py

```diff
@@ -99,8 +99,8 @@ def test_albef_image_embeddings_momentum(albef_model_output):
 def test_albef_text_embeddings(albef_model_output):
     expected = Tensor(
         [
-            [[-0.317956, 1.352367, -1.034411], [1.064044, -1.338780, 0.274735]],
-            [[-1.320019, 0.220507, 1.099512], [1.411497, -0.781628, -0.629869]],
+            [[-0.332726, 1.356729, -1.024002], [1.050448, -1.345235, 0.294787]],
```

Comment:
Noob question: why are the results changing here? Is the new encoder layer initializing its parameters in a different order?

Reply:
Yes I'm pretty sure it's just order but I will double check